project/scripts/fetch_trends.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+#!/usr/bin/env python3
+
+
+import pandas as pd
+from pytrends.request import TrendReq
+pytrends = TrendReq(tz=0) #tz=0 puts us on UTC
+
+from dates import get_start_times, get_end_times
+
+
+def get_updated_daily_data(query, investment_date):
+    """
+    query: a search to retrieve data for
+    investment_date: the date to start fetching data from as a string "yyyy-mm-dd"
+    returns a pandas dataframe of daily data for the query from the investment date to today
+    """
+    start = get_start_times(investment_date)
+    end = get_end_times()
+
+    hourly_data = fetch_hourly_data([query], *start, *end)
+    daily_data = aggregate_hourly_to_daily(query, hourly_data)
+
+    return daily_data
+
+
+def fetch_hourly_data(query, year_start, month_start, day_start, year_end, month_end, day_end):
+    """
+    queries: a search to retrieve data for
```

Comment:
queries in doc vs query in the arg list. if it is query, then update the comment about the return value

Reply:
I've reworked the variable naming throughout the code and docstrings.